### DISCUSSION TITLE
Ai and Voice assistant

### DISCUSSION THREAD

**Discussion by PARKKYUWON7777** [community/community#58205] [Accessibility]
### Select Topic Area

Question

### Body

접근성은 무한한 가능성을 열어줄거라고 생각합니다.
장애인을 위해 AI가 중요한 역할을 할수 있다고 봅니다. 그들도 개발자가 될 수있고 AI를 활용한다면 평범한 사람 이상의 감각과 창의력을 가지고 있다고 봅니다.

**Comment by ghostinhershell** [comment]
Hi @PARKKYUWON7777, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
